mailet/standard/pom.xml

```diff
@@ -142,6 +142,10 @@
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
```

Comment:
we try to keep our dependencies ordered (first by groupId then by artifactId). However we split dependencies in two categories: internal then externals.